python/ray/autoscaler/aws/defaults.yaml

```diff
@@ -111,8 +116,7 @@ setup_commands:
     # Note: if you're developing Ray, you probably want to create an AMI that
     # has your Ray repo pre-cloned. Then, you can replace the pip installs
     # below with a git checkout <your_sha> (and possibly a recompile).
-    - echo 'export PATH="$HOME/anaconda3/envs/tensorflow_p36/bin:$PATH"' >> ~/.bashrc
```

Comment:
Can you re-add this, but using the `tensorflow2_latest_p37` environment instead? I think it would be better for users to at least land in an environment with some other deps installed :)